src/client/utils/dom.js

```diff
@@ -347,7 +347,7 @@ export function isHammerheadAttr (attr) {
 }
 
 export function isIframeElement (el) {
-    return getTagName(el) === 'iframe';
+    return getTagName(el) === 'iframe' && isDomElement(el);
```

Comment:
May be we can to move the `isDomElement` condition into the `getTagName` function?